Jenkinsfile.promote

```diff
@@ -267,6 +285,18 @@ pipeline {
                 }
             }
         }
+
+        stage('Set Quickstarts next snapshot version') {
+            when {
+                expression { return isRelease() }
+            }
+            steps {
+                script {
+                    prNextSnapshotVersion(quickstartsRepository, getSnapshotBranch(), getBuildBranch());
+                    prNextSnapshotVersion(quickstartsRepository, "development", "development");
```

Comment:
shouldn't `development` be on the next `minor` version instead of micro ?
If yes, this should happen when release branch is created and not when new version is released I think.